lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -728,6 +728,11 @@ private function Annotation()
             }
         }
 
+        // strip the first \ off if used
```

Comment:
Can actually just call `ltrim($name, '\\')` all the time, and make this comment and conditional redundant.